synapse/storage/registration.py

```diff
@@ -92,6 +92,13 @@ def get_user_by_id(self, user_id):
             query, user_id
         )
 
+    def get_all_users(self):
+        query = ("SELECT users.name FROM users")
+        return self._execute(
+            self.cursor_to_dict,
+            query
+        )
```

Comment:
I tried to use ``_simple_select`` but bumped into problems because it assumed a ``WHERE`` clause. Is there a particular one I should be using?